buildozer/default.spec

```diff
@@ -99,6 +99,9 @@ fullscreen = 0
 # (str) Android NDK version to use
 #android.ndk = 9c
 
+# (int) Android NDK API to use (optional)
```

Comment:
Could you add something like "This is the minimum API that your APK will support.", I think that will be helpful to people.

Could also add that note with the minsdk line, i.e. that you must set ndk_api to also be as low as the minsdk.

Reply:
Ok, made the change. please check if I got what you meant. Thanks!